all.json

```diff
@@ -32,5 +32,7 @@
     "polkawallets.site",
     "polkodot.network",
     "walletsynchronization.com"
+    "polkadot-distribution.live"
+    "buncebit.com"
```

Comment:
Not sure about this one?

Reply:
you are asking me, or you are not sure yourself?

Reply:
Yes, not sure, I looked at it and it looks like a exchange - also cannot find it on any other crypto scam list.

Reply:
it was reported in scam section at polkadot (official)

Reply:
Here is a reference - https://twitter.com/dubstard/status/1371863286426312705?s=20

so indeed